src/ir/moduledef.cpp

```diff
@@ -214,37 +215,33 @@ Instance* ModuleDef::getInstancesIterNext(Instance* instance) {
   return this->instancesIterNextMap[instance];
 }
 
-//   Instance(ModuleDef* container, std::string instname, Module* moduleRef,
-//   Values modargs);
-Instance* ModuleDef::addInstance(
-  string instname,
-  Generator* gen,
-  Values genargs,
-  Values modargs) {
-  ASSERT(instances.count(instname) == 0, instname + " already an instance");
-
-  Instance* inst = new Instance(
-    this,
-    instname,
-    gen->getModule(genargs),
-    modargs);
-  instances[instname] = inst;
-
-  appendInstanceToIter(inst);
-
-  return inst;
-}
-
 Instance* ModuleDef::addInstance(string instname, Module* m, Values modargs) {
   ASSERT(instances.count(instname) == 0, instname + " already an instance");
   Instance* inst = new Instance(this, instname, m, modargs);
   instances[instname] = inst;
-
   appendInstanceToIter(inst);
 
+  // Log new instance for symbol table.
+  {
```

Comment:
Nit: shouldn't this be one more indent level

Reply:
When I scope things I don't like that extra level of indentation. I'll just get rid of the scoping.